examples/ble/scan.toit

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2023 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the examples/LICENSE file.
+
+import ble
+
+BATTERY_SERVICE ::= ble.BleUuid "180F"
+SCAN_DURATION   ::= Duration --s=3
+
+main:
```

Comment:
Strictly this is almost the same as the connect example. Maybe show some additional features of scan?